gpt4all-chat/modellist.h

```diff
@@ -187,9 +190,10 @@ class EmbeddingModels : public QSortFilterProxyModel
     Q_OBJECT
     Q_PROPERTY(int count READ count NOTIFY countChanged)
 public:
-    explicit EmbeddingModels(QObject *parent);
-    int count() const;
+    EmbeddingModels(QObject *parent, bool requireInstalled);
+    int count() const { return rowCount(); }
```

Comment:
ohh.... nevermind above comments about count. Although I don't know why you moved these to the header, but ok...

Reply:
I feel like if we are going to have *any* inline function definitions (and we have several), functions with one-word bodies are a good candidate.